.gitignore

```diff
@@ -69,6 +69,7 @@ amplify/backend/function/submissionsLambda/src/*.js
 amplify/backend/function/submissionsLambda/src/controllers
 public/**/**/nextImageExportOptimizer/
 public/next-image-export-optimizer-hashes.json
+src/directory/directory.json
```

Comment:
Added this so that it ignores all future changes, but I'm not sure if this is working as expected. Will be looking into this today